lib/components/user/wizard.tsx

```diff
@@ -95,9 +95,14 @@ class Wizard extends Component<Props> {
     } else {
       // Display a toast to acknowledge saved changes
       // (although in reality, changes quietly took effect in previous screens).
-      toastSuccess(
-        title,
-        intl.formatMessage({ id: 'actions.user.preferencesSaved' })
+      toastPromise(
+        // Fake promise to make sure toast is announced to screenreaders. See https://github.com/opentripplanner/otp-react-redux/pull/1400 for details.
+        new Promise((resolve) => setTimeout(resolve, 100)),
```

Comment:
You could probably resolve the promise immediately without a timer, e.g. 
```suggestion
        new Promise((resolve) => resolve()),
```

Reply:
TS needs an argument for `resolve` so I added `null`! Thanks